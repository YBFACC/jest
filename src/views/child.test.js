import Child from "./child";
import Enzyme, { shallow } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
Enzyme.configure({ adapter: new Adapter() });

test("Child", () => {
	const wrapper = shallow(<Child />);
	expect(wrapper.html()).toMatchSnapshot();
	wrapper.find("#antd-btn").simulate("click");

	expect(wrapper.html()).toMatchSnapshot();
});
