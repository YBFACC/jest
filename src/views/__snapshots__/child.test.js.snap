// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `"<div><button id=\\"antd-btn\\" type=\\"button\\" class=\\"ant-btn ant-btn-default\\"></button><span>dis-visible</span></div>"`;

exports[` 2`] = `"<div><button id=\\"antd-btn\\" type=\\"button\\" class=\\"ant-btn ant-btn-default\\"></button><span>visible</span></div>"`;

exports[`Child 1`] = `"<div><button id=\\"antd-btn\\" type=\\"button\\" class=\\"ant-btn ant-btn-default\\"></button><span>dis-visible</span></div>"`;

exports[`Child 2`] = `"<div><button id=\\"antd-btn\\" type=\\"button\\" class=\\"ant-btn ant-btn-default\\"></button><span>visible</span></div>"`;

exports[`renders learn react link 1`] = `"<div><button id=\\"antd-btn\\" type=\\"button\\" class=\\"ant-btn ant-btn-default\\"></button><span>dis-visible</span></div>"`;

exports[`renders learn react link 2`] = `"<div><button id=\\"antd-btn\\" type=\\"button\\" class=\\"ant-btn ant-btn-default\\"></button><span>visible</span></div>"`;
